test/fuzz/fuzz_test.cc

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.#pragma once
+
+#include "jwt_verify_lib/jwt.h"
+#include "jwt_verify_lib/jwks.h"
+#include "jwt_verify_lib/verify.h"
+
+#include "absl/strings/str_join.h"
+#include "absl/strings/str_split.h"
+
+#include <cstdint>
+#include <cstddef>
+#include <string>
+
+namespace google {
+namespace jwt_verify {
+namespace {
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+  // split the data int "jwt . jwks".
+  std::vector<std::string> v = absl::StrSplit(std::string((const char*)data, size), ".");
+  // jwt has 2 dot, at least we should have 3 dots with 4 sections.
```

Comment:
Too much validation? Fuzzer should pass these invalid inputs (JWTs that don't have 3 dots) to the JWT verify lib. It is realistic input the lib might receive

Reply:
Updated to use libprotobuf-mutator